src/http/response.ts

```diff
@@ -91,47 +66,13 @@ export class Response {
   //////////////////////////////////////////////////////////////////////////////
 
   /**
-   * Render html files. Can be used with Drash's template engine or basic HTML
-   * files. This method will read a file based on the `views_path` and filename
-   * passed in. When called, will set the response content type to "text/html"
+   * Used by Tengine:
    *
-   * @deprecated
-   * @param args - The arguments used to render.
-   *
-   * @remarks
-   *     // if `views_path` is "/public/views",
-   *     // file to read is "/public/views/users/add.html"
-   *     const content = this.response.render('/users/add.html', { name: 'Drash' })
-   *     if (!content) throw new Error(...)
-   *     this.response.body = content
-   *
-   * @returns The html content of the view, or false if the `views_path` is not
-   * set.
+   *   https://github.com/drashland/deno-drash-middleware/tree/master/tengine
    */
   public render(
     ...args: unknown[]
   ): Promise<boolean | string> | boolean | string {
```

Comment:
Should we not remove this method now?

Reply:
we need this so that users can still use this.response.render()

Reply:
Don't suppose you could add that as a note for the method?

Reply:
ah right

Reply:
78ae033d1b5b4bfae98f068ad0dbdf1a5c7c7073